packages/shared/ReactTypes.js

```diff
@@ -203,6 +203,20 @@ export type ReactEnvironmentInfo = {
   +env: string,
 };
 
+export type ReactErrorInfoProd = {
+  +digest: string,
+}
+
+export type ReactErrorInfoDev = {
+  +digest?: string,
```

Comment:
We currently don't transport the digest in error values. I don't see why we shouldn't.

Reply:
Might be awkward with `undefined` vs not defined and it didn't really come up yet anyway.